doc/cdi-operator-olm.md

```diff
@@ -92,6 +137,23 @@ cluster/kubectl.sh apply -f  _out/manifests/release/cdi-cr.yaml
 ```
 Now the operator should finish its deployment successfully
 
+<a name="update"></a>
+### CDI OLM update
+OLM mechanism supports operator update via subscription mechanism. Once subscription manifest is installed on cluster, it monitors the operator source and when new OLM bundle appears, OLM can trigger update of the operator.
+
+*Note:* Currently CDI operator does **not** support upgrades of the CDI installation, but it can be updated via OLM. In such case OLM update will effectivley terminate current _cdi-operaor_ instance and install the new one - specified in the new CSV bundle.
+
+#### Generate OLM bundle 
+Command ```make manifests``` fetches previous CSV_VERSION of CDI from  QUAY_REPOSITORY in QUAY_NAMESPACE  inorder to set it in *ReplacesVersion* field in new CSV manifest.
```

Comment:
Typo:
in order to set it.
inorder is two words.

Reply:
Done